Classes/Aspects/ContentCacheAspect.php

```diff
@@ -3,6 +3,8 @@
 
 namespace MOC\Varnish\Aspects;
```

Comment:
Namespace needs to be adjusted.

Reply:
@daniellienert the package is still in the MOC\Varnish namespace in 4.0. I'd like to release a `4.1.1` with this bugfix since Neos 5.X is still supported.

This patch will need to be upmerged to master later for Neos 7.X compatibility.

What do you think?

Reply:
You are so right ...  ☺️